lib/components/narrative/utils.tsx

```diff
@@ -19,7 +19,7 @@ export function localizeGradationMap(
   gradationMap?: Record<string, any>
 ): Record<string, any> {
   if (!gradationMap) return {}
-  const newGradationMap: Record<string, any> = {}
+  const newGradationMap: Record<string, any> = { ...gradationMap }
 
   Object.keys(newGradationMap).forEach((key) => {
```

Comment:
I would probably just change `newGradationMap` to `gradationMap` instead of copying `gradationMap` into the new object.

Reply:
I think that would modify the original object and I want to double check if we want to do that (although since you wrote this if that is what we want to do please let me know and I'll make the change)